Core/Object Arts/Dolphin/Base/IntegerTest.cls

```diff
@@ -68,6 +68,14 @@ testIsPowerOfTwo
 	powersOfTwo do: [:each | self assert: each isPowerOfTwo].
 	nonPowersOfTwo do: [:each | self deny: each isPowerOfTwo]!
 
+testIsPrime
+	| isPrimeResults |
+
+	isPrimeResults := #( 1 2 3 4 5 6 7 8 9 10 11 12 13 1000 7919 ) collect: [ :num | num isPrime ].
+
+	self assert: isPrimeResults = 
```

Comment:
This would provide more immediately actionable test failure information if each case were asserted individually, e.g:

```| cases |
cases := #((1 false) (2 true) (3 true) (4 false).... etc.
cases do: [:pair | self assert: pair first isPrime equals: pair second]```